docs/administration/support-package.md

```diff
@@ -0,0 +1,13 @@
+# Support Package
+
+Support packages are usefull for troubleshooting issue that occurs on OpenCTI platform.
+Administrators can request to create and download a support package that contains recent platform error logs and usage statistics.
+
+Support Package can be requested from "Settings > Support" menu.
+
+![Support package overview](./assets/support-package-overview.png)
+
+1. On a click on "Generate support package" button on the top right, the support package is requested.
+1. A background task on the platform gather logs and interesting data and create an archive

```

Comment:
A background task on the platform **gathers** logs and interesting data and **create** an archive

Reply:
```suggestion
1. A background task on the platform gathers logs and interesting data and creates an archive

```